### PR TITLE
Trim the expansion suffix off card names.

### PR DESCRIPTION
**Pull request by kberg**
Before
image

After
image

### CODE REVIEW

File: src/components/GameEnd.ts [hunk 1]
```diff
@@ -154,7 +154,8 @@ export const GameEnd = Vue.component('game-end', {
                                 <span :class="'log-player ' + getEndGamePlayerHighlightColorClass(p.color)"><a :href="'/player?id='+p.id+'&noredirect'">{{p.name}}</a></span>
                             </div>
                             <div v-for="v in p.victoryPointsBreakdown.detailsCards">
-                                {{v}}
+                                <!-- Removing any expansion-specific suffix-->
+                                {{v.split(':')[0]}}
```
Comment (nwai90) [inline src/components/GameEnd.ts:158]: Is it also intended to remove the `: VP_AMOUNT` after each card's name?

Comment (bafolts) [reply src/components/GameEnd.ts:158]: Guessing this wasn't intended. May need to strip this on the server.

Comment (kberg) [reply src/components/GameEnd.ts:158]: Whoops.

**Comment by kberg** [comment]
I've rewritten this to be a different kind of PR. Closing it until I test it satisfactorally manually.

**Pull request closed without merging**